kore/src/Kore/Log/InfoExecutionDepth.hs

```diff
@@ -48,15 +48,15 @@ instance Pretty InfoExecutionLength where
             , Pretty.pretty (getDepth depth)
             ]
 
-instance Entry InfoExecutionLength where
+instance Entry InfoExecutionDepth where
     entrySeverity _ = Info
     helpDoc _ = "log execution or proof length"
 
-infoProofLength
+infoProofDepth
     :: MonadLog log
     => [ProofState goal]
     -> log ()
-infoProofLength proofStateList =
+infoProofDepth proofStateList =
     case depthLongestProven proofStateList of
         Just n -> logEntry (LongestProvenClaim (ExecutionDepth n))
         _ -> forM_ (depthSomeUnproven proofStateList)
```

Comment:
The list can't actually contain any unproven nodes, because an unproven node throws an exception.